scripts/gceworker.sh

```diff
@@ -24,7 +24,7 @@ case ${1-} in
     sleep 20 # avoid SSH timeout on copy-files
 
     gcloud compute copy-files . "${name}:scripts"
-    gcloud compute ssh "${name}" "GOVERSION=${GOVERSION} ./scripts/bootstrap-debian.sh"
+    gcloud compute ssh "${name}" --ssh-flag="-A" --command="GOVERSION=${GOVERSION} ./scripts/bootstrap-debian.sh"
```

Comment:
here we're specifically just running a command, while `--` is used to just pass everything (flags, commands, who knows) though to `ssh` wth no handling .